src/Numerics/DeterminantOperators.h

```diff
@@ -232,6 +233,20 @@ inline typename MatA::value_type DetRatioByColumn(const MatA& Minv, const VecB&
   return simd::dot(Minv.cols(), Minv.data() + colchanged, Minv.cols(), newv.data(), 1);
 }
 
+/** determinant ratio with a column substitution
+ * @param Minv inverse matrix
+ * @param newv column vector
+ * @param colchanged column index to be replaced
+ * @return \f$ M^{new}/M\f$
+ */
+template<typename MatA, typename VecB>
+inline void mw_DetRatioByColumn(const int nw,std::vector<typename MatA::value_type> &curRatio_list, const RefVector<MatA>& Minv_list, const RefVector<VecB>& newv_list, int colchanged)
```

Comment:
The return value list (curRatio_list) should be at the end of the argument list.   The general pattern for these functions seems to be input arguments first, then output arguments.

Reply:
Agreed. Thanks for the function description though.